maps/graph_edges/base_graph/edges_dro.py

```diff
@@ -10,36 +10,36 @@
+    {"from": {"map": "DRO_01", "id": 0},           "to": {"map": "DRO_01", "id": "ShopItemA"}, "reqs": [["RF_MouserReturned"]]}, #* Outpost 1 Exit West -> ShopItemA (ThunderBolt)
     # {"from": {"map": "DRO_01", "id": 0}, "to": {"map": "DRO_01", "id": "ShopItemB"}, "reqs": []}, #* Outpost 1 Exit West -> ShopItemB (DustyHammer)
-    {"from": {"map": "DRO_01", "id": 0}, "to": {"map": "DRO_01", "id": "ShopItemC"}, "reqs": [["RF_MouserReturned"]]}, #* Outpost 1 Exit West -> ShopItemC (HoneySyrup)
+    {"from": {"map": "DRO_01", "id": "ShopItemA"}, "to": {"map": "DRO_01", "id": "ShopItemC"}, "reqs": []}, #+ SHARED REQUIREMENTS -> ShopItemC (HoneySyrup)
     # {"from": {"map": "DRO_01", "id": 0}, "to": {"map": "DRO_01", "id": "ShopItemD"}, "reqs": []}, #* Outpost 1 Exit West -> ShopItemD (DriedShroom)
     # {"from": {"map": "DRO_01", "id": 0}, "to": {"map": "DRO_01", "id": "ShopItemE"}, "reqs": []}, #* Outpost 1 Exit West -> ShopItemE (DriedPasta)
-    {"from": {"map": "DRO_01", "id": 0}, "to": {"map": "DRO_01", "id": "ShopItemF"}, "reqs": [["RF_MouserReturned"]]}, #* Outpost 1 Exit West -> ShopItemF (Mushroom)
+    {"from": {"map": "DRO_01", "id": "ShopItemA"}, "to": {"map": "DRO_01", "id": "ShopItemF"}, "reqs": []}, #+ SHARED REQUIREMENTS -> ShopItemF (Mushroom)
 
-    {"from": {"map": "DRO_01", "id": 0}, "to": {"map": "DRO_01", "id": 0}, "reqs": [["RF_MouserReturned"]], "pseudoitems": ["DriedPasta"]}, #+ ShopItemE
-    {"from": {"map": "DRO_01", "id": 0}, "to": {"map": "DRO_01", "id": 0}, "reqs": [["RF_MouserReturned"],["RF_CanUseDROCode"]], "pseudoitems": ["RF_CanMeetMoustafa"]}, #+ ShopItemE
-    {"from": {"map": "DRO_01", "id": 0}, "to": {"map": "DRO_01", "id": 0}, "reqs": [], "pseudoitems": ["RF_MouserLeftShop"]}, #+ ShopItemE
-    {"from": {"map": "DRO_01", "id": 0}, "to": {"map": "DRO_01", "id": 0}, "reqs": [], "pseudoitems": ["StarPiece_DRO_1",
-                                                                                                   #    "StarPiece_DRO_2",
-                                                                                                   #    "StarPiece_DRO_3",
-                                                                                                   #    "StarPiece_DRO_4",
-                                                                                                   #    "StarPiece_DRO_5",
-                                                                                                   #    "StarPiece_DRO_6",
-                                                                                                   #    "StarPiece_DRO_7",
-                                                                                                       "StarPiece_DRO_8"]}, #+ Quizmo StarPieces
+    {"from": {"map": "DRO_01", "id": "ShopItemA"}, "to": {"map": "DRO_01", "id": "ShopItemA"}, "reqs": [], "pseudoitems": ["DriedPasta"]}, #+ SHARED REQUIREMENTS -> ShopItemE
```

Comment:
Should be ShopItemE?

Reply:
Can't be ShopItemE, as E is unreachable from DRO_01/0 as it's commented out

Reply:
Right, I was misled by the comment

Reply:
I'm done reviewing, LGTM then